### ISSUE TITLE
more than one SharedState in ui-state directive?

### ISSUE THREAD

**Issue by martinstolz-ch** [mcasimir/mobile-angular-ui#200]
is it possible to define more than one SharedState in the ui-state directive like:

&lt;div class="tabs" ui-state="activeTab isAuth"&gt;

**Comment by mcasimir** [comment]
That's a good one, i'm going to add to the Roadmap.

In the meanwhile you can just use another ui-state somewhere else, just make sure it will be available once you need it.

**Comment by mcasimir** [comment]
Sorry i forgot to ask you.. what is `isAuth`? cause SharedState is intended to be a "shared" component state, and just hold UI logic. From my understanding of var name you're attempting to use it for application logic, something more suitable for a service:

```
ng-if="Auth.autenticated()"
```

I'm not suggesting anything here, I just wish to be sure that you were not forced to denaturate your app to meet the mobile angular ui syntax. In that case this would be a big problem and I should take care of it.

**Comment by martinstolz-ch** [comment]
you're right: 'isAuth' is not a good example for a shared state because it implies a mix bewteen business & presentation logic and thats truly not my intention. tx for your answers anyway!
